src/MapViewController.swift

```diff
@@ -17,12 +17,12 @@ import OnTheRoad
   case GeneralError, AnnotationDoesNotExist, APIKeyNotSet, RouteDoesNotExist
 }
 
-public class MapViewController: TGMapViewController, LocationManagerDelegate, TGRecognizerDelegate {
+public class MapViewController: UIViewController, LocationManagerDelegate, TGRecognizerDelegate {
 
   //Error Domains for NSError Appeasement
   public static let MapzenGeneralErrorDomain = "MapzenGeneralErrorDomain"
-
-
+  
+  var tgViewController: TGMapViewController = TGMapViewController()
```

Comment:
On Android `MapView` and `MapController` are separate objects whereas in iOS `TGMapViewController` embodies both of these classes in that it is used to load the map _and_ provide controls like position, zoom, and the marker interface.

There are few cases where access to the underlying `MapView` would be needed on Android so I could see changing this to be protected in that developers could subclass `MapView` if they really needed access.

Exposing the `MapController` on Android and `TGMapViewController` on iOS makes more sense from the perspective that a developer may want to customize the behavior of the map in certain situations or have access to functionality that is not currently supported by the SDK.